test/e2e/simple_preemptor/simple_preemptor_suite_test.go

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
Named it based on the original file - simple_preemptor.go. Also, "preemptor" has been named similarly like scheduler etc. If you still think otherwise, we should do even for original files and can be taken separately in a new jira.